swift/llm/sft.py

```diff
@@ -215,7 +251,7 @@ def llm_sft(args: SftArguments) -> Dict[str, Union[str, Any]]:
         logging_first_step=True,
         **kwargs)
 
-    if args.gradient_checkpointing:
+    if args.gradient_checkpointing and not use_torchacc():
```

Comment:
Write the modifications for gradient_checkpointing in the argument.py directly.

Reply:
done